api/v1/hull.js

```diff
@@ -0,0 +1,11 @@
+import { buildConvexHull } from '@jsxcad/algorithm-shape';
+import { CSG } from './CSG';
+
+export const hull = (...geometries) => {
+  var allPoints = [];
+  for (const geometry of geometries) {
+    allPoints = allPoints.concat(geometry.toPoints());
```

Comment:
This will work, but it will copy the data in allPoints a bunch of times.

You could pipe eachPoint through, or if that seems a bit much, consider

const allPoints = [].concat(...geometries.map(geometry => geometry.toPoints));

as an alternative.

Reply:
const allPoints = [].concat(...geometries.map(geometry => geometry.toPoints()));

rather